pkg/api/volume_req.go

```diff
@@ -51,6 +51,7 @@ type VolOptionReq struct {
 type VolOptionResetReq struct {
 	Options []string `json:"options"`
 	Force   bool     `json:"force,omitempty"`
+	All     bool     `json:"all,omitempty"`
```

Comment:
When `All` is specified, user doesn't have to provide `Options` right ? Doesn't that mean `Options` can be `omitempty` ? If that's the case, server or CLI has to validate that as follows:

* If `All` is present, ignore `Options`
* If `All` is not present, `Options` should be present.